simplified-profiles/src/main/java/org/nypl/simplified/profiles/ProfileDescriptionJSON.kt

```diff
@@ -408,7 +407,11 @@ object ProfileDescriptionJSON {
     )
 
     return map.mapValues { entry ->
-      PlayerSleepTimerConfiguration.valueOf(entry.value)
+      try {
+        entry.value.toLong()
+      } catch (exception: NumberFormatException) {
+        null
```

Comment:
Does this mean that if there's an unexpected/non-numeric value in the json, the sleep timer will be set to end-of-file? I think it should be set to be off if there is an unexpected value. You may need to check for null explicitly before calling `toLong`, in order to check for end-of-file.

Also, from the test, it looks like older versions of the app were saving values like "MINUTES_15" and "MINUTES_30", right? If so, I think we need to check for these as well, for backward compatibility.

Reply:
@nunommts Yes, you're right. If restoring the sleep timers wasn't working before, then I don't think we need to worry about handling those old values.